packages/transformer/src/IModelTransformer.ts

```diff
@@ -1943,17 +1932,21 @@ export class IModelTransformer extends IModelExportHandler {
       : undefined;
 
     if (this._options.preserveElementIdsForFiltering) {
-      const isInvalid = !Id64.isValid(targetElementId);
+      // initialize set if undefined to indicate in importer that preserveElementIdsForFiltering option is true
+      if (!this.importer.elementIdsToUpdateForPreserveId) {
```

Comment:
How do you feel about doing this right after we construct an instance of the importer? (or if we're given one too)

Reply:
yeah, that will a better place to add it

Reply:
Added